kore/src/Kore/Implicit/ImplicitSorts.hs

```diff
@@ -12,124 +12,28 @@ module Kore.Implicit.ImplicitSorts where
 
 import Kore.AST.MetaOrObject
 import Kore.Implicit.ImplicitSortsImpl
-import Kore.MetaML.AST
 import Kore.Sort
 
 -- TODO(virgil-serbanuta): Add tests for "defined but not used" symbols
 charMetaSort     :: Sort Meta
 charListMetaSort :: Sort Meta
-nilCharList      :: MetaSentenceSymbol
-nilCharListA     :: MetaPatternStub
-consCharList     :: MetaSentenceSymbol
-consCharListA    :: [MetaPatternStub] -> MetaPatternStub
-appendCharList   :: MetaSentenceSymbol
-appendCharListA  :: [MetaPatternStub] -> MetaPatternStub
-inCharList       :: MetaSentenceSymbol
-inCharListA      :: [Sort Meta] -> [MetaPatternStub] -> MetaPatternStub
-deleteCharList   :: MetaSentenceSymbol
-deleteCharListA  :: [MetaPatternStub] -> MetaPatternStub
-charListAxioms   :: [MetaSentenceAxiom]
-( charMetaSort, charListMetaSort
-    , nilCharList, nilCharListA, consCharList, consCharListA
-    , appendCharList, appendCharListA
-    , inCharList, inCharListA
-    , deleteCharList, deleteCharListA
-    , charListAxioms
-    )
-  =
-    defineMetaSort CharSort
+(charMetaSort, charListMetaSort) = defineMetaSort CharSort
```

Comment:
I think that these two are the only sorts that we need right now (and the String alias).

Reply:
I think you are right, but some of the tests still use the other meta sorts for now.